proposals/dao/tip_110.ts

```diff
@@ -0,0 +1,109 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc,
+  PcvStats
+} from '@custom-types/types';
+import { getImpersonatedSigner, overwriteChainlinkAggregator, ZERO_ADDRESS, balance, time } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+
+let pcvStatsBefore: PcvStats;
+let daiBalanceBefore: BigNumber;
+
+const fipNumber = '110';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  // Deploy Lido stETH/USD oracle
+  const angleEuroRedeemerFactory = await ethers.getContractFactory('AngleEuroRedeemer');
+  const angleEuroRedeemer = await angleEuroRedeemerFactory.deploy();
+  await angleEuroRedeemer.deployed();
+  logging && console.log(`angleEuroRedeemer: ${angleEuroRedeemer.address}`);
+
+  return {
+    angleEuroRedeemer
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  // make sure ETH oracle is fresh (for B.AMM not to revert, etc)
+  // Read Chainlink ETHUSD price & override chainlink storage to make it a fresh value
+  const ethPrice = (await contracts.chainlinkEthUsdOracleWrapper.read())[0].toString() / 1e10;
+  await overwriteChainlinkAggregator(addresses.chainlinkEthUsdOracle, Math.round(ethPrice).toString(), '8');
+
+  // read pcvStats before proposal execution
+  pcvStatsBefore = await contracts.collateralizationOracle.pcvStats();
+
+  // angle multisig action : make enough USDC collateral available for redemptions
+  const angleMultisigSigner = await getImpersonatedSigner('0x0C2553e4B9dFA9f83b1A6D3EAB96c4bAaB42d430');
+  await forceEth(angleMultisigSigner.address);
+  await contracts.anglePoolManagerUsdc.connect(angleMultisigSigner).updateStrategyDebtRatio(
+    addresses.angleStrategyUsdc1, // USDC strategy has 57M deployed
+    '0'
+  );
+  await contracts.angleStrategyUsdc1.harvest();
```

Comment:
This multisig action has to be done by the Angle multisig, coordinate with them so that at least 9.6M USDC is available for redemptions at time of execution